settings.py

```diff
@@ -154,4 +154,12 @@ def insert_debug_toolbar_middleware(middlewares):
 SWAGGER_SETTINGS = {
     'USE_SESSION_AUTH': False,
     'DEEP_LINKING': True,
+    'SECURITY_DEFINITIONS': {
+        'Session': {
```

Comment:
Can we add support for JWT auth too?  https://addons-server.readthedocs.io/en/latest/topics/api/auth.html or is that not possible?  Thinking: we write these docs to be used by 3rd party developers, and JWT auth is what we encourage for them (because it doesn't require a browser login to get the session id, and doesn't expire)

Reply:
It's a little bit _odd_. So actually all this does is add a UI box to authenticate adding an `Authorization` header. It will let you put `JWT <token>` but I can see how the naming is a bit confusing.

I updated it to have session and jwt as separate options to make it clear there are both.
I added descriptions to both, and included how to format the header.

Does that help?

<img width="754" alt="image" src="https://github.com/mozilla/addons-server/assets/19595165/39d0c0da-92f5-47b3-9d8f-a0959565bceb">